hello_gadget/README.md

```diff
@@ -0,0 +1,72 @@
+# hello_gadget
+
+This is an example of using
+[nerves_init_gadget](https://github.com/fhunleth/nerves_init_gadget) to
+initialize a simple application that runs on a board supporting USB gadget mode.
+Currently this is limited to the Raspberry Pi Zero and Raspberry Pi Zero W.
+
+## Getting started
+
+This image supports firmware updates using `nerves_firmware_ssh`. To use this
+feature, the SSH public keys that are allowed to update the firmware need to be
+added to `config/config.exs`. This process is similar to adding a key to an
+`authorized_keys` file on a server. See the `config/config.exs` for where to put
+your key.
+
+Then, in this directory, run the following to build and program a MicroSD card:
+
+```
+MIX_TARGET=rpi0 mix deps.get
+MIX_TARGET=rpi0 mix firmware
+
+# Insert a MicroSD Card into a card reader and "burn" it
+MIX_TARGET=rpi0 mix firmware.burn
+```
+Now plug the MicroSD card into a Raspberry Pi Zero. Connect your laptop or
+computer to the Raspberry Pi Zero's USB port that's labeled "USB" (not the one
```

Comment:
I can't bring myself to fix this one, but I fixed all of the other comments. Thanks for catching those.